grant-project/grant-monorepo
# Team invite email

Further progresses #143.

### What This Does

Finishes off the email sending behavior from #220. Inviting a user to the team who has an email available now sends them one. The email has two displays, users who have accounts and users who do not yet.

### Steps to Test

1. Invite a user to team by email, confirm it sends
2. Invite an existing user to team by their address, confirm it sends
3. Invite a non-existing user to team by their address, confirm it doesn't send

## Screenshots

### Existing user

<img width="873" alt="screen shot 2018-12-03 at 7 17 38 pm" src="https://user-images.githubusercontent.com/649992/49410090-55d0b700-f731-11e8-92ea-0ecc5054cf44.png">

### New user

<img width="870" alt="screen shot 2018-12-03 at 7 13 09 pm" src="https://user-images.githubusercontent.com/649992/49410101-5cf7c500-f731-11e8-933f-f2b0e744bf2c.png">
